mimesis/helpers.py

```diff
@@ -18,10 +20,9 @@ def randints(self, amount: int = 3, a: int = 1, b: int = 100) -> list:
         if not amount:
```

Comment:
I feel like we should restrict this further now that I'm looking at it; e.g., the predicate could probably be `amount == 0`, and in an `else` clause `raise ValueError('Amount out of range')` or something.